integration_tests/shell.nix

```diff
@@ -24,4 +24,8 @@ mkShell {
       nixpkgs-fmt
       cosmovisor
     ];
+  shellHook = ''
+    # prefer local pystarport directory for development
+    export PYTHONPATH=$PWD/pystarport:$PWD/pystarport/pystarport/proto_python:$PYTHONPATH
```

Comment:
I think you don't need this for ci testing

Reply:
other autogenerated python code need to `proto_python` path as a module,
also to run python module elsewhere, it's better to specify absolute path in shell.nix

Reply:
I guess you can config import path to the tools?

Reply:
programatically adding path?

Reply:
i'll check protoc option